### PR TITLE
Conversation block: rename to Transcription block

### PR DESCRIPTION
**Pull request by creativecoder**
#### Changes proposed in this Pull Request:

* Renames the "Conversation" block to "Transcription", including the folder name and any references to "conversation" in code
* Renames "Participant" to "Speaker" in the block UI, to be consistent (but all code reference continue to reference "participant")

The PR looks like a lot of changes, but everything is simply renaming, along with Prettier formatting updates.

#### Does this pull request change what data or activity we track or use?

No

#### Testing instructions:

* Insert a _transcription_ block, and verify adding speakers and dialogue works as before
* Search the code and make sure no references to "conversation" remain (other than keywords)

#### Proposed changelog entry for your changes:

* Conversation block: rename to "Transcription"

### CODE REVIEW

File: projects/plugins/jetpack/extensions/shared/icons.js [hunk 1]
```diff
@@ -168,18 +168,6 @@ export const DonationsIcon = {
...
-	src: (
-		<SVG viewBox="0 0 24 24">
-			<Rect x="0" fill="none" width="24" height="24" />
-			<G>
-				<Path d="M15.9216 2H2.98533C2.43803 2 1.99023 2.45 1.99023 3V17L5.97062 13H15.9216C16.4689 13 16.9167 12.55 16.9167 12V3C16.9167 2.45 16.4689 2 15.9216 2ZM14.9265 4V11H5.14469L3.98043 12.17V4H14.9265ZM18.9069 6H20.8971C21.4444 6 21.8922 6.45 21.8922 7V22L17.9118 18H6.96572C6.41842 18 5.97062 17.55 5.97062 17V15H18.9069V6Z" />
-			</G>
-		</SVG>
-	),
-};
-
```
Comment (creativecoder) [inline projects/plugins/jetpack/extensions/shared/icons.js:182]: This is the previous conversation block icon--we are no longer using it.

**Comment by creativecoder** [comment]
cc @sfougnier who requested this (p1612899054275500-slack-C01FD4FGD7X)

> Conversation > Transcript, Dialogue > Transcript Speaker

This renames the conversation block to "Transcript", but uses "Transcript Dialogue" for the child blocks, rather than "Transcript Speaker". But all the UI labels of "Participant" have been changed to "Speaker". Semantically, this made more sense to me (the block belongs to a speaker, but it's content is still dialogue).

**Review by retrofox — COMMENTED** [review]
I'd like to give my opinion on renaming these blocks.
A _Transcription_ is always tied to a _source_. And on this context, it means tied to the audio of a podcast, but it’s possible to use the Conversation block without any source.

For example, a user might create a short conversation between two people on their blog. That isn’t a transcription as we understand in this context. Let's keep in mind these blocks will be published on Jetpack sites.

On other hand, Dialogue is much more than a transcription, basically because it defines the person who speaks its content (Speaker). And finally, it fits perfectly with the idea of a Conversation block wrapping a lot of Dialogue's blocks to create a conversation. So far in this comment, I haven't mentioned a podcast or another source.

A transcript could be just a paragraph block with the content defined by the source (podcast episode, generic audio, a video, ...), a quote block, etc. Also, it would be possible to extend this paragraph block to be able to add links to the media source to be able to playback at its position. This is a transcription but is not a dialogue.

My two coins.

**Comment by retrofox** [comment]
Wondering when this decision came from. [We discussed renaming blocks a long time ago](p1606251307158600-slack-) to anticipate and prevent this issue. Iterate and re-iterate is part of the dev process, so it isn't a big deal either, just wondering why.

**Comment by gamebits** [comment]
Can this block be used for a single-speaker monologue or podcast? In which case, that's neither a conversation nor a dialogue.

**Comment by creativecoder** [comment]
> A Transcription is always tied to a source. And on this context, it means tied to the audio of a podcast, but it’s possible to use the Conversation block without any source.

That's a great point @retrofox ! Conversation does seem like a more semantically correct name for the block, given it can be used on it's own.

> Can this block be used for a single-speaker monologue or podcast? In which case, that's neither a conversation nor a dialogue.

@gamebits It could, but the block assumes dialogue from multiple speakers to be very useful. Though this points out a weakness in the block for the single-speaker use case. You'd still have to create multiple dialogue entries to mark timestamps, even if the speaker didn't change, so the same speaker name would repeat.

---

I'm going to remove the 9.4.1 milestone, as it seems like we don't have time to resolve this discussion before that release.

**Comment by retrofox** [comment]
> I'm going to remove the 9.4.1 milestone, as it seems like we don't have time to resolve this discussion before that release.

Thanks, and I'm sorry about that.

**Comment by creativecoder** [comment]
I'm going to close this, as it seems like keeping the block names as is will work for now. If we decide this is important in the future, we can update and reopen this PR.

**Pull request closed without merging**